pkg/services/hivelocity/client/mock/mock_client.go

```diff
@@ -100,6 +100,18 @@ var NoTagsDevice = hv.BareMetalDevice{
 	OsName:      defaultImage,
 }
 
+// MultiTagsDevice is a device which has multiple tags.
```

Comment:
maybe MultiLabelsDevice???

Reply:
```go
var MultiTagsDevice = hv.BareMetalDevice{
	Hostname: "host3-unused",
	Tags: []string{
		"caphvlabel:foo1=bar1",
		"caphvlabel:foo2=bar2",
	},
	DeviceId:    NoTagsDeviceID,
	PowerStatus: "ON",
	OsName:      defaultImage,
}
```

from code we still use these labels as tags in the machine, so I think MultiTag is more reasonable?

Reply:
together with NoTagsDeviceID? No i think labels makes more sense, to distinguish it. and "caphvlabel:" shows also that is is tags in form of labels.